common/djangoapps/third_party_auth/migrations/0002_samlproviderconfig_country.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.14 on 2020-07-22 22:45
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('third_party_auth', '0001_squashed_0026_auto_20200401_1932'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='samlproviderconfig',
+            name='country',
+            field=models.CharField(blank=True, help_text=('URN of SAML attribute containing the user`s country.',), max_length=128),
```

Comment:
wohoo! this helps for sure explain what this should be

Reply:
and 128 because?

Reply:
See this marked as resolved but for clarity, 128 because I wanted to follow the pattern of the other urns in the model. Open to suggestions if you think it's not quite what we're looking for. I'm unsure as to how long an average urn is/what the max length is.